daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -373,6 +381,31 @@ object Node {
     type WithTxValue[+Nid, +Cid] = F[Nid, Cid, Transaction.Value[Cid]]
   }
 
+  case class VersionedNode[+Nid, +Cid](
+      version: TransactionVersion,
```

Comment:
The term "transaction version" will soon stop making sense, won't it? When that time has come, we should probably rename the type.

Reply:
:+1:

Reply:
I will keep `TransactionVersion` as long `ValueVersion` is not killed.